pkg/cli/scan.go

```diff
@@ -637,81 +616,194 @@ func resolveInputFileFromArg(ctx context.Context, tmpdir, inputFilePath string)
 // For example, given the input value "calico", this function will find the
 // latest version of the package (e.g. "calico-3.26.3-r3.apk") and download it
 // for each architecture.
-func resolveInputForRemoteTarget(ctx context.Context, input string) (downloadedAPKFilePaths []string, cleanup func() error, err error) {
+func resolveInputForRemoteTarget(ctx context.Context, indices map[string]map[string]*apk.APKIndex, arch, input string) (string, error) {
```

Comment:
Can we update the function doc comment to describe the new way this function works?